signing_clients/apps.py

```diff
@@ -385,7 +398,8 @@ def issuer(self):
     def signer_certificate(self):
         for certificate in self.content['certificates']:
             info = certificate['tbs_certificate']
-            if info['issuer'] == self.issuer and info['serial_number'] == self.signer_serial_number:
+            if info['issuer'] == self.issuer and \
+               info['serial_number'] == self.signer_serial_number:
```

Comment:
Not a fan of multi-line if statements. I'd prefer something like

```python
is_signer_certificate = (
    info['issuer'] == self.issuer and
    info['serial_number'] == self.signer_serial_number)

if is_signer_certificate:
    return info
```

which is way easier to read imho.